app/src/main/java/net/opendasharchive/openarchive/services/dropbox/DropboxSiteController.kt

```diff
@@ -231,9 +232,14 @@ class DropboxSiteController : SiteController {
                 projectName
             )
             if (getUseProofMode()) {
+                var metaMediaHash = ProofMode.generateProof(mContext, Uri.parse(media.originalFilePath),media.mediaHashString)
+                if(metaMediaHash == null){
```

Comment:
Is leaving no spaces at all Kotlin formatting standards? Highly doubt it. Space between letters improves readability.

Reply:
Incredibly, the formatter did not bother to change this for readibility despite running it multiple times.. Just added in a space manually for now. It could be my local installation that is messed up for all I know.